igniter/bootstrap_repos.py

```diff
@@ -668,9 +668,9 @@ def empty_progress(x: int):
         self._progress_callback = progress_callback
 
         if getattr(sys, "frozen", False):
-            self.live_repo_dir = Path(sys.executable).parent / "repos"
+            self.live_repo_dir = Path(sys.executable).parent
```

Comment:
I think we should remove the use of `self.live_repo_dir` as it has no effect now. That probably involves slight refactor of `create_version_from_live_code()`

Reply:
Changed it to use `OpenPypeVersion.get_installed_version()` which seems to be easiest way how to get all needed information.

You can do changes into branch if you'd like to...